docs/tools/devtools.mdx

```diff
@@ -50,6 +50,24 @@ Eg.
 }
 ```
 
+Example for Vite + React project:
+
+```ts
+// vite.config.ts
+
+export default defineConfig({
+  plugins: [
+    react({
+      babel: {
+        plugins: [
```

Comment:
Isn't it presets instead of plugins? Does it work with this?

Reply:
No, my bad. Not familiar at all with babel. Fixed it now